examples/transform/quip_example.py

```diff
@@ -22,7 +22,7 @@
 #   * quantize the weights to 4 bit with a group size 128
 recipe = [
     QuIPModifier(
-        rotations=["v", "u"], transform_block_size=128, transform_type="random-hadamard"
+        rotations=["v", "u"], transform_block_size=128, transform_type="hadamard"
```

Comment:
I think this folder is missing a readme explaining the examples with snippets on how to run this in vLLM.